security/BLAKE3/Portfile

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cargo   1.0
+PortGroup           github  1.0
+
+github.setup        BLAKE3-team BLAKE3 1.3.1
+github.tarball_from archive
```

Comment:
can we have the name not capitalized?

Reply:
Hi @reneeotten,

I have made this change in the Portfile.

Thanks,

-ranga